packages/pagination/src/PaginationControls.js

```diff
@@ -53,7 +55,11 @@ const PaginationControls = ({
 
   const createBreak = (index) => (
     <PaginationItem key={index} data-testid={`control-page-${index}`}>
-      <PaginationLink onClick={() => handleBreakClick(index)} type="button">
+      <PaginationLink
+        onClick={() => handleBreakClick(index)}
+        type="button"
+        aria-label="ellipsis"
```

Comment:
What is the ellipsis used for? We should have the `aria-label` reflect that rather than just calling it out as an ellipse